lib/rspec/core/bisect/fork_runner.rb

```diff
@@ -92,7 +92,6 @@ def initialize(runner, channel)
 
           def dispatch_specs(run_descriptor)
             pid = fork { run_specs(run_descriptor) }
```

Comment:
Need to not capture pid if this is the change we're making

Reply:
Yep! Thanks